llpc/CMakeLists.txt

```diff
@@ -144,16 +144,21 @@ set_compiler_options(llpc ${LLPC_ENABLE_WERROR})
 
 ### Defines/Includes/Sources ###########################################################################################
 if(ICD_BUILD_LLPC)
-    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${XGL_LLVM_BUILD_PATH}/lib/cmake/llvm)
+    list(APPEND CMAKE_MODULE_PATH
+        ${XGL_LLVM_BUILD_PATH}/lib/cmake/llvm
+        ${XGL_LLVM_BUILD_PATH}/${CMAKE_CFG_INTDIR}/lib/cmake/llvm   # Workaround for multi-config generators (e.g. VS)
+    )
```

Comment:
I think we should only include one if these paths. `CMAKE_CFG_INTDIR` is an empty string for non-multi-config generators, so the second path should work there as well.

Reply:
True, it would work for single- and multi-config generators now, but I decided to add it as a separate path in case LLVM is fixed at some point. If that happens, `CMAKE_CFG_INTDIR` in the path would break VS generator. Having it in path doesn't make sense, so I'd rather have it separately so it's obvious this is a kind of a hack.

Reply:
Having read https://cmake.org/cmake/help/latest/variable/CMAKE_CFG_INTDIR.html, I tend to agree with @yavn: we really shouldn't be using CMAKE_CFG_INTDIR here if everything was working as it is supposed to. The only reason for doing so is to workaround a bug elsewhere.